raiden/api/rest.py

```diff
@@ -176,13 +197,23 @@ def get_tokens_list(self):
         result = self.tokens_list_schema.dumps(tokens_list)
         return result
 
-    def get_new_events(self):
-        raiden_service_result = self.get_new_events()
-        assert isinstance(raiden_service_result, list)
+    def get_network_events(self, from_block, to_block):
+        raiden_service_result = self.raiden_api.get_token_added_events(
+            from_block, to_block
+        )
+        return normalize_events_list(raiden_service_result)
 
-        events_list = EventsList(raiden_service_result)
-        result = self.events_list_schema.dumps(events_list)
-        return result
+    def get_token_events(self, token_address, from_block, to_block):
```

Comment:
This is very misleading, `get_token_events` reads as "Events emitted by the Token smart contract" whereas in reality this endpoint returns the `ChannelManager` `TokenAdded` event, perhaps renaming it to `get_channel_events` or `get_token_network_events`?